lib/modules/manager/fleet/extract.ts

```diff
@@ -0,0 +1,107 @@
+import { loadAll } from 'js-yaml';
+import { regEx } from '../../../util/regex';
+import { GitTagsDatasource } from '../../datasource/git-tags';
+import { HelmDatasource } from '../../datasource/helm';
+import { checkIfStringIsPath } from '../terraform/util';
+import type { PackageDependency, PackageFile } from '../types';
+import type { FleetFile, FleetFileHelm, GitRepo } from './types';
+
+function extractGitRepo(doc: GitRepo): PackageDependency {
+  const dep: PackageDependency = {
+    depType: 'git_repo',
+    datasource: GitTagsDatasource.id,
+  };
+
+  const repo = doc.spec?.repo;
+  if (!repo) {
+    return {
+      ...dep,
+      skipReason: 'missing-depname',
+    };
+  }
+  dep.sourceUrl = repo;
+  dep.depName = repo;
+
+  const currentValue = doc.spec.revision;
+  if (!currentValue) {
+    return {
+      ...dep,
+      skipReason: 'no-version',
+    };
+  }
+
+  return {
+    ...dep,
+    currentValue,
+  };
+}
+
+function extractFleetFile(doc: FleetFileHelm): PackageDependency {
+  const tempDep: PackageDependency = {
```

Comment:
```suggestion
  const dep: PackageDependency = {
```

like in other function